Jenkinsfile

```diff
@@ -145,7 +101,7 @@ pipeline {
                   }
                 }
               }
-              stage('Windows'){
+              stage('Windows .NET Framework'){
                 agent { label 'windows-2016' }
```

Comment:
if we split the thing we'd use `windows` because we have only one windows-2016 agent, doing that, we'd use the windows-2012 too

Reply:
IIUC, the label `windows-2016` should avoid using `windows-2012`, if so, I don't understand what's the issue here

Reply:
we only have one windows 2016 worker with one executor, so it only can run a task at a time, if we change the label to `windows` we will use both windows workers.

Reply:
Got it, thanks for the clarification, let's keep it sequentially for now, likely we might need to see if we can create ephemeral windows workers somehow if possible.